tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```

Comment:
**question**: What type of import does this allow? A quick search brought up this example but I'm not sure if we have any cases of this:

```typescript
import * as React from 'react'; // sad face
```

source: [Understanding Typescript: "allowSyntheticDefaultImports": true](https://gist.github.com/chrisgervang/59ed046c0a8d7c3a1be1b3416f8f2466)

Reply:
Yeah this setting makes it so that we don't _have_ to do the `import * as` dance, so for example this line can remain the same, instead of the sad face version above:

https://github.com/artsy/rosalind/blob/23d94afa77a13f25cbc69cb6c59122bcdb9226e5/app/javascript/components/batch_update/components/App.js#L1

Reply:
Ahh, I see. Thanks for clarifying 👍🏼